src/libraries/Microsoft.Extensions.Diagnostics.Abstractions/src/Metrics/MeterFactoryExtensions.cs

```diff
@@ -32,7 +31,7 @@ public static Meter Create(this IMeterFactory meterFactory, string name, string?
             {
                 Version = version,
                 Tags = tags,
-                Scope = scope
+                Scope = meterFactory
```

Comment:
If a factory is always going to set the Meter's scope to itself, and if DefaultMeterFactory effectively ignores Scope, do we actually need it on MeterOptions?

Reply:
Yes, we need it for the scenario of users not using DI or the factory.

Reply:
For the Meter ctor? Ok.

Reply:
It's used here on `Meter`: https://github.com/dotnet/runtime/blob/84ad0d38173f104524cd6b743c50ca0d1d92b1c6/src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/Meter.cs#L24-L36

`MeterOptions` doesn't exactly fit both situations: direct ctor vs `IMeterFactory.Create`.